specs/_features/epbs/fork-choice.md

```diff
@@ -150,7 +150,7 @@ def block_for_inclusion_list(store: Store, inclusion_list: InclusionList) -> Opt
     
     blocks = blocks_for_slot(store, summary.slot)
     for block in blocks:
-        if block.slot == summary.slot and block.proposer_index == summary.proposer_index and block.signed_execution_payload_header.message.parent_block_hash == parent_hash:
```

Comment:
I dont think you need the slot check. Please correct me if I'm wrong here. `blocks_for_slot(store, summary.slot)` already covers that

Reply:
Nice one! thanks